### PR TITLE
DApp-1690 dashboard analytics with stakingpools

### PR DESCRIPTION
**Pull request by rohitmalhotra1420**
### Ticket Number

- [1690](https://github.com/push-protocol/push-dapp/issues/1690)

### Description

- Implemented the notification, message and subscriber count on /welcome page. These counts matches the push.network numbers.
- Implemented the staking pools section on /welcome page.

- **Problem/Feature**:

### Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other (please describe):

### Checklist

- [x] **Quick PR**: Is this a quick PR? Can be approved before finishing a coffee.
  - [ ] Quick PR label added
- [ ] **Not Merge Ready**: Is this PR dependent on some other PR/tasks and not ready to be merged right now.
  - [ ] DO NOT Merge PR label added

### Frontend Guidelines

- [ ] Followed frontend guidelines as per the [Guidelines Notion Doc](https://www.notion.so/pushprotocol/Frontend-dApp-Guidelines-1d7806ae3d9e4569a340b563dcd0536c)

### Build & Testing

- [x] No errors in the build terminal
- [x] Engineer has tested the changes on their local environment
- [x] Engineer has tested the changes on deploy preview

### Screenshots/Video with Explanation

- **Before:** Explain the previous behavior

- **After:** What's changed now

### Review & Approvals

- [x] Self-review completed
- [ ] Code review by at least one other engineer
- [ ] Documentation updates if applicable

### CODE REVIEW

**Review by corlard3y — COMMENTED** [review]
Screenshot 2024-07-29 at 21 01 33

Just noticed the Total Value Locked formatting not working like the rest of the figures.

**Comment by rohitmalhotra1420** [comment]
> Screenshot 2024-07-29 at 21 01 33
> 
> Just noticed the Total Value Locked formatting not working like the rest of the figures.

Fixed it @corlard3y

**Review by mishramonalisha76 — CHANGES_REQUESTED** [review]
image
The description doesnot look like the one in figma, needs to be a lesser width.
Also a little bit more gap between the skeleton and the apr, staked, would look better.
The values keeps on loading and i get the exception division by zero. I think its coming intermittently.

File: src/queries/models/analytics/getSubscriberCountModelCreator.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+): number => {
+  const subscriberAnalyticsData = response?.subscriberAnalytics;
+  let totalSubscribers = 0;
+  const dateArray = getDatesArray({
+    start: dates.start,
+    end: dates.end,
+    interval: 1,
+  });
+
+  let subscriberArray: any[] = [];
```
Comment (mishramonalisha76) [inline src/queries/models/analytics/getSubscriberCountModelCreator.ts:30]: can we add proper type to the array inspite of any

**Review by mishramonalisha76 — APPROVED** [review]

**Pull request merged**